### PR TITLE
Added the states column in the participants view

### PR DESCRIPTION
**Pull request by nikhilgupta1211**
A person can only have a single active request for an event. The State column shows the state of the active request of a user.

### CODE REVIEW

**Comment by nikhilgupta1211** [comment]
image

File: app/helpers/events_helper.rb [hunk 1]
```diff
@@ -16,4 +16,19 @@ def users_for_event(state)
     requests = requests.where(state: state) if state != 'all'
     requests.distinct.pluck('users.email')
   end
+
+  def state_label(state)
```
Comment (ChrisBr) [inline app/helpers/events_helper.rb]: Just return the css class and do the rest in the view because it is always the same

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def participants
       { label: @event.name, url: event_path(@event) },
       { label: 'participants' }
     ]
-    @requests = @event.travel_sponsorships.includes(:user).select(:user_id).distinct.accessible_by(current_ability)
+    @requests = @event.travel_sponsorships.includes(:user).group(:user_id).accessible_by(current_ability)
```
Comment (ChrisBr) [inline app/controllers/events_controller.rb]: Why is this change necessary?

File: app/helpers/events_helper.rb (hunk 1 above)
Comment (nikhilgupta1211) [inline app/helpers/events_helper.rb]: ok

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (nikhilgupta1211) [inline app/controllers/events_controller.rb]: In the previous query we are only selecting the user_id from a request for distinct requests if we also select state here then the requests will not be distinct, also group returns the latest request of the user(active request) I don't know if distinct also do the same.

File: app/helpers/events_helper.rb [hunk 3]
```diff
@@ -16,4 +16,19 @@ def users_for_event(state)
...
+    case state
+    when 'submitted'
+      'label label-primary'
```
Comment (nikhilgupta1211) [inline app/helpers/events_helper.rb]: yes ok :)
